packages/p2p/src/components/verification/verification.jsx

```diff
@@ -53,7 +54,7 @@ const Verification = ({ should_wrap }) => {
 
     return (
         <VerificationWrapper should_wrap={should_wrap}>
-            <div className='dp2p-verification'>
+            <div className='dp2p-verification' data-testid='component_container'>
```

Comment:
What do you think about using more specific naming for `testid`? 
suggested : `data-testid='dp2p-verification_container'`

Reply:
> What do you think about using more specific naming for `testid`? suggested : `data-testid='dp2p-verification_container'`

we are already in p2p package, so it's questionable :)

Reply:
> What do you think about using more specific naming for `testid`? suggested : `data-testid='dp2p-verification_container'`

but 'verification_container is definitely better, thx for suggestion! :)